front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ProjectsWidget/Settings.tsx

```diff
@@ -19,18 +21,19 @@ const Settings = () => {
 
   const {
     actions: { setProp },
-    publicationStatus,
+    publicationStatuses,
   } = useNode((node) => ({
-    publicationStatus: node.data.props.publicationStatus || 'published',
+    publicationStatuses: node.data.props.publicationStatuses || ['published'],
```

Comment:
```suggestion
    publicationStatuses: node.data.props.publicationStatuses?.length ?  node.data.props.publicationStatuses : ['published'] : 
```

maybe we can add something ike this so that the 'published' is at least shown by default?